foolbox/models/pytorch.py

```diff
@@ -60,9 +60,13 @@ def batch_predictions(self, images):
         images = torch.from_numpy(images)
         if self.cuda:  # pragma: no cover
             images = images.cuda()
-        images = Variable(images, volatile=True)
-        predictions = self._model(images)
-        predictions = predictions.data
+        if torch.__version__[:3] < '0.4':
```

Comment:
This version number comparison could break in the future (e.g. version 0.10.0; given the announcement of v1.0, this will probably never be the case, but still…)